material-overrides/assets/stylesheets/tanssi.css

```diff
@@ -615,6 +615,25 @@ li>nav.md-nav[data-md-level="4"] {
   display: none
 }
 
+/* Disclaimer styling */
+.md-content__inner .main-page {
+  display: flex;
+  flex-direction: column;
+  min-height: 75vh;
+  margin-bottom: 0.5rem;
+}
+
+.disclaimer,
+.page-disclaimer {
+  margin-top: auto;
+  font-size: x-small;
+  font-style: italic;
```

Comment:
this styling already exists for the `.disclaimer` class and there is no `.page-disclaimer` in the tanssi repo  -- line 666